r2-core/src/main/java/com/linkedin/r2/transport/http/client/AsyncPoolStats.java

```diff
@@ -64,6 +66,8 @@ public AsyncPoolStats(
       int totalDestroyErrors,
       int totalBadDestroyed,
       int totalTimedOut,
+      int totalWaiterTimedOut,
+      int totalCreationIgnored,
```

Comment:
totalCreationsIgnored
or to be even more explicit
totalCreationsCancelledNoWaiter

Reply:
I will vote for totalCreationsIgnored